### DISCUSSION TITLE
I propose to move the server to PVP mode.

### DISCUSSION THREAD

**Discussion by SQWIZI1337** [chromiecraft/chromiecraft#4128] [Suggestions]
1) This will add more content for max level players, most players will be able to run around the world without fear of being killed. This will make the world more lively in terms of pvp.
2) For low level players this update will also be a nice bonus as they will have more events.

**Comment by BeefSteaker** [comment]
Forced world PvP I'm not in favour of, but wouldn't mind specific zones being made pvp

**Comment by 55Honey** [comment]
Hello @SQWIZI1337 ,
Thanks for your suggestion.
It was brought up before countless times. The stance of ChromieCraft is, that we will not force players into world PvP.

So regardless of any support, the answer to that was and will remain no.

**Reply by SQWIZI1337** [reply]
World pvp is not coercion

**Reply by 55Honey** [reply]
It is though if you can not unflag yourself, which inevitably happens when a server moves from PvE to PvP setting.
